network/OpenSSL/OpenSSLConnection.hpp

```diff
@@ -105,6 +105,8 @@ namespace awsiotsdk {
             bool certificates_read_flag_;
             bool enable_alpn_;
 
+            bool ipv6_connection;                       ///< Connect using ipv6
```

Comment:
This member is never initialized anywhere, and should be named ```ipv6_connection_``` to match the coding standard. That said, it might actually be more semantically straightforward to do this by storing the address family as a member instead, since most reads of this member are translated into ```AF_INET``` or ```AF_INET6```.

Reply:
I reworked the PR with the suggested change to use the address family instead of a boolean.